README.md

```diff
@@ -0,0 +1,508 @@
+# openHAB Cloud
+
+openHAB Cloud is a companion cloud service and backend for the openHAB open-source home automation software.
+The openHAB Cloud backend provides secure remote access and enables openHAB users to remotely monitor,
+control and steer their homes through the internet, collect device statistics of their openHABs, receive
+notifications on their mobile devices or collect and visualize data etc.
+The main core features of openHAB-cloud are an user-management frontend, secure remote access, remote proxy-access, device registry & management, messaging services and data management & persitence.
+The openHAB Cloud also serves as core backend integration point for cloud-based features (e.g. IFTTT) and
+provides an OAuth2 application enablement.
+
+
+## Funtional Architecture ##
+
+
+
+![FunctionalArchitecture_openHAB-cloud_v1.0.png](https://bitbucket.org/repo/XkeoB7/images/3807118397-FunctionalArchitecture_openHAB-cloud_v1.0.png)
+
+
+
+## Getting started
+
+
+openHAB Cloud is mainly based on the following frameworks and technologies:
+
+
+* [Node.js](https://nodejs.org/en/) - Server-side Javascript-framework
+* [Express.js](http://redis.io) - Web application framework for Node.js
+* [Nginx](https://www.nginx.com/resources/wiki/) - Web server & reverse proxy
+* [MongoDB](https://www.mongodb.com/) - NoSQL database
+* [redis](http://redis.io) - Session Manager & data structure server
+* [Socket.IO](http://socket.io/) - Bi-directional communication between web clients and servers
+* [MEAN](https://en.wikipedia.org/wiki/MEAN_(software_bundle)) - Software stack
+
+
+### Quick start ###
+
+Installing openHAB Cloud on Linux (ubuntu).
+
+
+First we need to make sure that the list of packages and dependencies
+from all repositories are up-to-date: 
+
+```
+sudo apt-get update
+sudo apt-get upgrade
+sudo apt-get dist-upgrade
+```
+
+If you got asked to continue, confirm with Yes (Y).
```

Comment:
"If you are asked to continue" , I think you can also run these with "-y", alternatively we could leave this out, if you can't figure this out, you should not be installing openHAB Cloud ;-)